x/ibc/03-connection/types/connection.go

```diff
@@ -0,0 +1,60 @@
+package types
+
+import (
+	ics23 "github.com/cosmos/cosmos-sdk/x/ibc/23-commitment"
+)
+
+// ICS03 - Connection Data Structures as defined in https://github.com/cosmos/ics/tree/master/spec/ics-003-connection-semantics#data-structures
+
+// ConnectionState defines the state of a connection between two disctinct
+// chains
+type ConnectionState = byte
+
+// available connection states
+const (
+	NONE ConnectionState = iota // default ConnectionState
```

Comment:
Love this syntax! There are a number of other spots we should be using this.

Reply:
Cute method of 1-indexing :)